packages/components/src/templates/next/components/internal/GoogleTagManager/utils.ts

```diff
@@ -0,0 +1,10 @@
+export const shouldNotIncludeGoogleTagManager = (): boolean => {
```

Comment:
nit: maybe we can simplify to directly assign the conditionals to the variable rather than use a function here? For example:

```ts
export const SHOULD_INCLUDE_GTM = typeof process !== "undefined" && process.env.NODE_ENV === "production"
```

If we switch to `site.environment` would be more straightforward too.

```ts
export const SHOULD_INCLUDE_GTM = site.environment === 'production'
```